### ISSUE TITLE
[Question] Which repo is used by "Create Branch" in a multi repo workspace?

### ISSUE THREAD

**Issue by fvet** [tfenster/azdevops-vscode-simplify#37]
We work a lot with workspace files containing **multiple repo's** / apps.

When I use the 'Create branch' shortcut in the simplify menu bar on a certain work item, how does the extension identify in which repo to create the branch?

I only had one active editor with a file from our 'Core' repo, which pointed to the master branch, although after using 'Create branch', a branch was made in **another** other (wrong) repo instead?

![image](https://user-images.githubusercontent.com/25268332/218998183-386e0de2-bfe5-4da6-9a48-bc84c4dd9b4e.png)

**Comment by tfenster** [comment]
@fvet Sorry for the late reply, I didn't see this. At the moment, this is hard-coded to the first remote fetchurl returned by the VS Code Git extension (see https://github.com/tfenster/azdevops-vscode-simplify/blob/main/src/api/azdevops-api.ts#L600). This is not a great implementation in multi-repo scenarios, but worked for @DavidFeldhoff and me when we created this, so we decided to leave it at that. We might change that in the future, but also feel free to provide a PR with a better implementation :)
